test/Datadog.Trace.Security.IntegrationTests/AspNetCore5.cs

```diff
@@ -22,6 +22,8 @@ public AspNetCore5(ITestOutputHelper outputHelper)
         [Theory]
         [InlineData(true, HttpStatusCode.Forbidden)]
         [InlineData(false, HttpStatusCode.OK)]
+        [Trait("RunOnWindows", "True")]
+        [Trait("Category", "ArmUnsupported")]
```

Comment:
Is the appsec stuff in general not supported on ARM? or is it something about this test?

Reply:
We don't yet have an arm build of the C++ Sqreen Library that does actually analysis, so we can't support it yet. I checked on the status of this the other day, the code cross compiles, it just needs releasing. So, support is coming soon.

Reply:
Cool, thanks 👍